temporal-sdk/src/main/java/io/temporal/worker/WorkflowImplementationOptions.java

```diff
@@ -126,13 +129,39 @@ public Builder setDefaultLocalActivityOptions(
       return this;
     }
 
+    /**
+     * Set individual Nexus Service options per service. Will be merged with the map from {@link
+     * io.temporal.workflow.Workflow#newNexusServiceStub(Class, NexusServiceOptions)} which has the
+     * highest precedence.
+     *
+     * @param nexusServiceOptions map from service to NexusServiceOptions
+     */
+    public Builder setNexusServiceOptions(Map<String, NexusServiceOptions> nexusServiceOptions) {
```

Comment:
No need to update doc, but just for me to understand, map key is service name here not class name right (even though class name is default service name)?